ansible_base/resource_registry/models/service_id.py

```diff
@@ -0,0 +1,27 @@
+import uuid
+
+from django.db import models
+
+
+class ServiceID(models.Model):
+    """
+    Provides a globally unique ID for this service.
+    """
+
+    id = models.UUIDField(default=uuid.uuid4, primary_key=True, null=False, editable=False)
```

Comment:
Services do have a name right? In the RBAC branch I introduced a setting `ANSIBLE_BASE_SERVICE_PREFIX` which defaults to "local" for the test_app mainly. I fully expect that apps need to declare a setting that gives their service name like "awx". It seems like that would get added to the ServiceID model.

We're not supporting multiple services of the same type, still fine to have the uuid, but even if we were that's not inconsistent with tracking the name as an informational property. Right now the name only gets added in the ResourceType.name combined with other strings, but I am absolutely sure the service name part by itself is going to get reused for other things.